omni-relayer/src/main.rs

```diff
@@ -96,7 +80,17 @@ async fn main() -> Result<()> {
     handles.push(tokio::spawn({
         let config = config.clone();
         let redis_client = redis_client.clone();
-        async move { startup::evm::start_indexer(config, redis_client).await }
+        async move { startup::evm::start_eth_indexer(config, redis_client).await }
+    }));
+    handles.push(tokio::spawn({
+        let config = config.clone();
+        let redis_client = redis_client.clone();
+        async move { startup::evm::start_base_indexer(config, redis_client).await }
+    }));
+    handles.push(tokio::spawn({
+        let config = config.clone();
+        let redis_client = redis_client.clone();
+        async move { startup::evm::start_arb_indexer(config, redis_client).await }
```

Comment:
It would be even better if we had a map from ChainKind to a function. This way, we could see which ChainKind values are defined and call the appropriate indexers.